src/main/archetype/ui.permissions/src/main/content/jcr_root/home/groups/__appsFolderName__/custom-administrators/.content.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:rep="internal"
+    jcr:mixinTypes="[rep:AccessControllable]"
+    jcr:primaryType="rep:Group"
+    jcr:uuid="9ec293ba-08f0-3d39-9db9-7ff85e311cb5"
+    rep:authorizableId="custom-administrators"
+    rep:principalName="custom-administrators">
```

Comment:
@gabrielwalt What would be the best way to associate this custom AEM administrator to the OOTB Administrators group automatically?

Reply:
What is the purpose of separating custom-administrators from administrators? Also, would it be more idiomatic to use the name "${appsFolderName}-administrator"?

Reply:
I agree with DG here. Administrator actions should be managed with the OOTB `administrators` group, otherwise you have separate management of permissions which may create security  governance problems down the road.